tests/lib/eslint.js

```diff
@@ -2437,6 +2437,9 @@ describe("eslint", function() {
 
         describe("Custom parser", function() {
 
+            var path = require("path");
+            var parserFixtures = path.join(path.dirname(module.id), "../fixtures/parsers") + "/";
```

Comment:
I don't know whether this will work correctly on Windows. Anyone have a better idea?